api/manifest.proto

```diff
@@ -65,6 +65,13 @@ message Build {
       LINUX_GLIBC = 3;
    }
 
+   enum Architecture {
```

Comment:
maybe add a comment (and possibly an upstream envoy link) of the source of the architecture choices? Knowing what this is derived from can help in wondering what the future choices might be also, as for example some call "arm64" "aarch64" and also sometimes there are qualifiers. like arm64v8.. similar for x86 vs amd64